src/__tests__/paxfulapi_refresh_test.ts

```diff
@@ -56,15 +61,19 @@ describe("With the Paxful API SDK", function () {
         fetchMock.once({
             name: 'correct_access_token',
             url: /oauth2\/userinfo/,
-            method: "GET"
+            method: "GET",
+            headers: {
+                Authorization: 'Bearer abc'
```

Comment:
I see that below we're veryfing that fetchMock is being called 3 times, e.g.:
`expect(fetchMock).toHaveFetchedTimes(3);`

But, does it verify that 
`paxfulApi.get(GET_PROFILE_URL)`
indeed at first fails and then eventually calls this endpoint with refreshed Bearer token, i.e. "abc"?

Reply:
Yep, above we are configuring all required asserts for every request including failing ones.
And this toHaveFetchedTimes(3) makes sure no more requests have been emitted except those 3.